main/reference/zoe-api/zcfmint.md

```diff
@@ -13,7 +13,7 @@ Returns an **IssuerRecord** containing the **[Issuer](/reference/ertp-api/issuer
 **[Brand](/reference/ertp-api/brand.md)** associated with the **zcfMint**.
 
 ## aZCFMint.mintGains(gains, zcfSeat?)
-  - **gains**: **[AmountKeywordRecord](./zoe-data-types.md#amountkeywordrecord)**
+  - **gains**: **[AmountKeywordRecord](./zoe-data-types.md#keywordrecord)**
```

Comment:
Why this change?

I don't see any `#keywordrecord` anchor in https://docs.agoric.com/reference/zoe-api/zoe-data-types.html

Reply:
It's being added by this PR: https://github.com/Agoric/documentation/pull/876/files#diff-d7fc746f4ec4970ac29b349e9549ae68d4152a1c2e982580b3e1a1cef1ed6e9aR66

> ## KeywordRecord
> 
> A **KeywordRecord** is a [CopyRecord](/glossary/#copyrecord) in which every property name
> is a **[Keyword](#keyword)**, such as `harden({ Currency: quatloosBrand, CurrencyUnit: 100n })`.
> Subtypes further constrain property values (for example, an
> **AmountKeywordRecord** is a **KeywordRecord** in which every value is an
> **[Amount](/reference/ertp-api/ertp-data-types.md#amount)** and a
> **PaymentKeywordRecord** is a **KeywordRecord** in which every value is a
> **[Payment](/reference/ertp-api/payment.md)**).
> 
> Users submit their **payments** and receive their **payouts** as **KeywordRecords**:
> ```js
> quatloosPurse.deposit(payouts.Asset);
> const quatloos5 = AmountMath.make(quatloosBrand, 5n);
> const payments = {
>   Asset: quatloosPurse.withdraw(quatloos5),
> };
> ```